content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -20,6 +20,15 @@ deprecated API versions to newer and more stable API versions.
 
 ## Removed APIs by release
 
+### v1.27
```

Comment:
removed in 1.26

Reply:
updated PR for `autoscaling/v2beta2` removal in 1.26